gemini/src/main/java/com/techempower/gemini/GeminiApplication.java

```diff
@@ -1324,12 +1307,8 @@ protected void runShutdownTasks()
         }
         catch (Exception exc)
         {
-          // In the event a shutdown tasks throws an exception, we will report
-          // the exception to stderr (because the log may be closed) and then
-          // proceed to the next shutdown task.
```

Comment:
At least in logback, the flushing of logs on app shutdown should be handled for you automatically. http://logback.qos.ch/manual/configuration.html#webShutdownHook

Reply:
Also, on http://logback.qos.ch/manual/appenders.html:
> By default, each log event is immediately flushed to the underlying output stream. This default approach is safer in the sense that logging events are not lost in case your application exits without properly closing appenders. However, for significantly increased logging throughput, you may want to set the `immediateFlush` property to false.